src/components/tweet-link.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import {sample} from 'lodash'
+import {displayListify} from '../../shared/utils'
+import shortenEpisodeUrl from '../../other/shorten-episode-url/logic'
+
+export default TweetLink
+
+function addRandomAwesomeEmojiTo(message) {
```

Comment:
I wonder if we could move this to `shared/utils` and re-use that logic for the `get-up-tweet` where this code came from originally :-)